blog/content/post/2020-12-14-trenchboot_attestation.md

```diff
@@ -0,0 +1,266 @@
+01:05:37 INFO  src/util/coap_util.c:51: Adding CoAP FETCH resource 'attest'.
+01:05:37 INFO  src/attester.c:101: [attester] Waiting for connections.
+01:06:06 INFO  src/attester.c:129: [attester] Resource 'attest': Received message.
+01:06:06 INFO  src/attester.c:140: [attester] Received data of length 63.
+01:06:06 INFO  src/attester.c:145: [attester] Parsing received CBOR data.
+01:06:06 INFO  src/attester.c:155: [attester] Preparing TPM quote data.
+01:06:06 INFO  src/attester.c:184: [attester] Loading TPM key.
+01:06:06 INFO  src/core/charra_key_mgr.c:36: Loading key "PK.RSA.default".
+01:06:19 INFO  src/util/tpm2_util.c:128: Primary Key created successfully.
+01:06:19 INFO  src/attester.c:193: [attester] Do TPM Quote.
+01:06:19 INFO  src/attester.c:196: [attester] sig_key_handle: 0x40418487
+01:06:19 INFO  src/attester.c:197: [attester] public_key: 1
+01:06:19 INFO  src/attester.c:203: [attester] TPM Quote successful.
+01:06:19 INFO  src/attester.c:209: [attester] Preparing response.
+01:06:19 INFO  src/attester.c:218: [attester] Marshaling response.
+01:06:19 INFO  src/attester.c:224: [attester] Adding marshaled data to CoAP response PDU and send it.
+01:06:19 INFO  src/attester.c:101: [attester] Waiting for connections.
+```
+
+[![asciicast](https://asciinema.org/a/7wp9hEjmj8iAPN1fjVPcYwWji.svg)](https://asciinema.org/a/7wp9hEjmj8iAPN1fjVPcYwWji)
+
+
+**Verifier**
+
+```
+01:06:06 INFO  src/verifier.c:84: [verifier] Starting up.
+01:06:06 INFO  src/verifier.c:93: [verifier] Initializing CoAP endpoint.
+01:06:06 INFO  src/verifier.c:104: [verifier] Registering CoAP resource handlers.
+01:06:06 INFO  src/verifier.c:108: [verifier] Creating new attestation request.
+01:06:06 INFO  src/verifier.c:131: [verifier] Marshaling attestation request data.
+01:06:06 INFO  src/verifier.c:145: [verifier] Adding attestation request data to CoAP PDU.
+01:06:06 INFO  src/verifier.c:153: [verifier] Sending CoAP message.
+01:06:19 INFO  src/verifier.c:251: [verifier] Resource 'attest': Received message.
+01:06:19 INFO  src/verifier.c:264: [verifier] Received data of length 1258.
+01:06:19 INFO  src/verifier.c:268: [verifier] Parsing received CBOR data.
+01:06:19 INFO  src/verifier.c:291: [verifier] Starting verification.
+01:06:19 INFO  src/verifier.c:301: [verifier] Loading TPM key.
+01:06:19 INFO  src/verifier.c:305: [verifier] External public key loaded.
+01:06:19 INFO  src/verifier.c:313: [verifier] Preparing TPM quote verification.
+01:06:19 INFO  src/verifier.c:322: [verifier] Verifying TPM Quote signature.
+01:06:19 INFO  src/verifier.c:327: [verifier] TPM Quote signature valid!
+01:06:19 INFO  src/verifier.c:345: [verifier] +----------------------------+
+01:06:19 INFO  src/verifier.c:346: [verifier] |   ATTESTATION SUCCESSFUL   |
+01:06:19 INFO  src/verifier.c:347: [verifier] +----------------------------+
+```
+
+[![asciicast](https://asciinema.org/a/8cAV0h2FTCghAEFZ09you3J0A.svg)](https://asciinema.org/a/8cAV0h2FTCghAEFZ09you3J0A)
+
+
```

Comment:
Maybe add a list of all contributions done as part of this work in on of the last sections.

Reply:
Added.